Grades.java

```diff
@@ -10,9 +10,54 @@
  * 2. The lowest grade and its index (something like: the student #3 got the lowest grade and it's 13).
  * 3. The average of the grades
  */
+
+package hw2arrays;
+
+import java.util.Scanner;
+
 public class Grades {
 
-    public static void main(String[] args) {
-        // enter your code here :)
-    }
+	public static void main(String[] args) {
+		Scanner in = new Scanner(System.in);
+		
+		System.out.print("Enter number of students:");
+		String outMessage;
+		byte numOfStud = in.nextByte();
+		byte[] grades = new byte[numOfStud]; 
+		
+		for (byte i = 0; i < grades.length; i++) {
+			System.out.print("Enter grade for student #" + i + ":");
+			byte tmpGrade = in.nextByte();
+			if(tmpGrade >= 0 && tmpGrade <= 100) {
+				grades[i] = tmpGrade;
+			} else {
+				System.out.println("Error, please enter number between 0 and 100!");
+				i--;
```

Comment:
It's considered a VERY bad practice to change the index inside an index loop.
The better solution would be to enter a while loop after the input from the user. For example,
      byte tmpGrade = in.nextByte();
       while(tmpGrade < 0 || tmpGrade > 100) {
	        System.out.println("Error, please enter number between 0 and 100!");
                tmpGrade = in.nextByte();
       }

Reply:
You are absolutely right! This is the first thing I wanted to do, but then I thought about optimization..  :)